src/util/Util.js

```diff
@@ -174,6 +174,51 @@ class Util {
         return webpMedia;
     }
 
+    /**
+     * Formats a media to ogg/opus
+     * @param {MessageMedia} media
+     * 
+     * @returns {Promise<MessageMedia> media} in opus/ogg (audio) format
+     */
+    static async formatToOpusAudio(media) {
+        if (!media.mimetype.includes('audio'))
+            throw new Error('media is not an audio');
+
+        if (media.mimetype === 'audio/ogg; codecs=opus')
```

Comment:
Maybe a failsafe for all oggs codecs or waweb only like opus?

Reply:
This is the only WA accepted audio format if you want it to be played in the application, all other formats will be converted. Do you have a different sugestion? I'm keeping as is.